### PR TITLE
Remove "total" from "total population" and some other stat var descriptions.

### PR DESCRIPTION
**Pull request by shifucun**
Consolidated and cleaned up some other stat vars.
Also updated svdiff tool README.

https://storage.mtls.cloud.google.com/datcom-embedding-diffs/boxu_base_uae_mem_2024_06_28_20_09_29.html

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
Thanks!

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 1]
```diff
@@ -3541,7 +3541,7 @@ WHO/mslv_Rural,Measles Immunization Coverage Among 1 year old in Rural Areas
 WHO/mslv_Urban,Measles Immunization Coverage Among 1 year old in Urban areas
 WHO/NCD_BMI_18A,Prevalence Of Underweight Among Adults
 WHO/NCD_BMI_30A,Prevalence Of Obesity Among Adults
-WHO/NCD_PAC,Prevalence Of Insufficient Physical Activity Among Adults
```
Comment (pradh) [inline tools/nl/embeddings/input/base/sheets_svs.csv:3544]: Not sure insufficient physical activity is same as physical inactivity?

Based on https://screenshot.googleplex.com/AM63bQgSndLPMjf

File: tools/nl/embeddings/input/base/sheets_svs.csv [hunk 2]
```diff
@@ -1842,7 +1842,7 @@ Count_MortalityEvent_HeartDiseaseExcludingHypertension,Number of Deaths from Hea
 Count_MortalityEvent_IntentionalSelf-Harm(Suicide),Number of Intentional Self-Harm (Suicide) Deaths
 Count_MortalityEvent_IntentionalSelf-Harm(Suicide)_Female,Number of Intentional Self-Harm (Suicide) Deaths in Females
 Count_MortalityEvent_Suicide,Number of Suicide Deaths
-Count_Person,total population
+Count_Person,population count
```
[the remark this answers is not in the document]
Comment (pradh) [reply tools/nl/embeddings/input/base/sheets_svs.csv:1845]: For instance, wonder if a redundant "count" throws this off?  - https://screenshot.googleplex.com/3sJnoRz8qXUqUqa

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 1 above)
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:3544]: reverted this

File: tools/nl/embeddings/input/base/sheets_svs.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (shifucun) [reply tools/nl/embeddings/input/base/sheets_svs.csv:1845]: I have tried "population" which is even worse.. it is too strong and matches too many things. So when a query asks about "population A" and stat vars like "population A+B" and "population A+B+C" could match lower than "population"

**Review by pradh — APPROVED** [review]

**Pull request merged**